examples/C++/HelloWorldExampleDS/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
```suggestion
# Copyright 2020 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Reply:
HelloWorldExampleDS does not belong (yet) into fastdds.